timed/employment/tests/test_user.py

```diff
@@ -213,3 +214,61 @@ def test_user_without_employment(self):
         assert result['data']['attributes']['worktime-balance'] == (
             '00:00:00'
         )
+
+    def test_user_absence_types(self):
+        credits = AbsenceCreditFactory.create_batch(3, user=self.user)
+
+        url = reverse('user-detail', args=[
+            self.user.id
+        ])
+
+        res = self.client.get('{0}?include=user_absence_types'.format(url))
+
+        assert res.status_code == HTTP_200_OK
+
+        result = self.result(res)
+
+        rel = result['data']['relationships']
+        inc = result['included']
+
+        assert len(rel['user-absence-types']['data']) == len(credits)
+        assert len(inc) == len(credits)
+
+        assert (
+            inc[0]['id'] ==
+            '{0}-{1}'.format(self.user.id, credits[0].absence_type.id)
+        )
+
+        assert type(inc[0]['attributes']['credit']) == int
+        assert type(inc[0]['attributes']['used-days']) == int
+        assert type(inc[0]['attributes']['balance']) == int
+        assert inc[0]['attributes']['used-duration'] is None
```

Comment:
We need to test whether the attributes have been calculated correctly. Instead of random data some fixed data would make this easily possible.